Assets/_BForBoss/_Core/RenderPipeline/UniversalRenderPipelineAsset_Renderer.asset

```diff
@@ -44,6 +44,20 @@ MonoBehaviour:
   m_ShadowTransparentReceive: 1
   m_RenderingMode: 0
   m_AccurateGbufferNormals: 0
+--- !u!114 &2160553602708355395
```

Comment:
What are these changes for?

Reply:
Oh these are definitely residue from when I was experimenting with the example that you [originally](https://github.com/MirzaBeig/Anime-Speed-Lines) sent. Unity was having some trouble deleting some of the CustomPostProcessing from the pipeline asset. It probably shouldn't have been pushed. I'll delete it 👍

Reply:
git command to revert these changes are 
`git checkout develop -- Asset/_BForBoss/_Core/RenderPipeline..../UniversalRenderPipelineAsset_Renderer.asset`